spec/openapi.yaml

```diff
@@ -775,6 +779,67 @@ paths:
         500:
           $ref: '#/components/responses/Error500ResponseDefinition'
 
+
+#
+# Peoples API
+#
+
+  /people-api/v2/users:
+    post:
+      summary: Get list of user information
+      tags:
+      - Peoples
+      description: >
+        People is user of the system who process, translate context
```

Comment:
Smartling users, who process and translate content with the Smartling platform.

Reply:
It was already done.